config/default_records/tag_sets/default_records.wip.yml

```diff
@@ -0,0 +1,13 @@
+---
+PN-1000215/PN-3000431 Dual Index Kit TT, Set A:
+  name: PN-1000215/PN-3000431 Dual Index Kit TT, Set A
+  tag_group_name: PN-1000215/PN-3000431 Dual Index Kit TT, Set A (i7)
+  tag2_group_name: PN-1000215/PN-3000431 Dual Index Kit TT, Set A (i5)
+PN-1000250 Dual Index Kit TN, Set A:
+  name: PN-1000250 Dual Index Kit TN, Set A
+  tag_group_name: PN-1000250 Dual Index Kit TN, Set A (i7)
+  tag2_group_name: PN-1000250 Dual Index Kit TN, Set A (i5)
+PN-1000251 Dual Index Kit TS, Set A:
+  name: PN-1000251 Dual Index Kit TS, Set A
+  tag_group_name: PN-1000251 Dual Index Kit TS, Set A (i7)
+  tag2_group_name: PN-1000251 Dual Index Kit TS, Set A (i5)
```

Comment:
Tag sets are completely hidden and there is no interface to access them for the moment. I don't think we need to WIP the file name as the next story y24-055 needs them but that is OK it can be enabled with that story too.

As a side note, I think, naming a WIP file with a generic name has a problem. We need to do `WIP=default_records bundle exec rake application:post_deploy` however it enables all WIP files with the name default_records. I would name it differently:
- name the file with a sequence number to show an order
- specific name to show the intention, for example 001_chromium_dual_index_tag_sets.wip.yml

Reply:
Yeah I think thats a valid point, I wasn't sure on the conventions for record loader but your naming makes sense, ill push a change.